src/compile/data/parse.ts

```diff
@@ -100,6 +104,16 @@ export function parseData(model: Model): DataComponent {
   // the current head of the tree that we are appending to
   let head = root;
 
+  const lookups = model.transforms.filter(isLookup);
```

Comment:
Hmm, this is bad. We want to preserve the order. I see that this is a problem...